src/components/Error/DAONotFoundError.js

```diff
@@ -1,76 +1,91 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import styled from 'styled-components'
-import { theme, Button } from '@aragon/ui'
-import ErrorCard from './ErrorCard'
+import {
+  useTheme,
+  textStyle,
+  ButtonText,
+  EmptyStateCard,
+  GU,
+  Info,
+} from '@aragon/ui'
 import { network } from '../../environment'
 import { isAddress } from '../../web3-utils'
+import notFoundImage from '../../assets/dao-not-found.png'
+
+function DAONotFoundError({ dao }) {
+  const theme = useTheme()
+  return (
+    <EmptyStateCard
+      css={`
+        display: grid;
+        grid-template-rows: 1fr auto auto auto;
+        grid-template-columns: 1fr;
+        padding: ${2 * GU}px;
+        width: ${72 * GU}px;
+        height: auto;
+        box-shadow: 0px 2px 4px rgba(180, 188, 202, 0.5);
+      `}
+      illustration={
+        <img
+          src={notFoundImage}
+          alt="Organization not found"
+          css={`
+            width: 147px;
+            height: 144px;
```

Comment:
Could we set these as props on the element? That way browser engines can do optimizations like only keeping the 1x version in memory.